src/Apps/Order/Components/ShippingSummaryItem.tsx

```diff
@@ -38,10 +40,17 @@ const ShippingSummaryItem = ({
           business days to coordinate pickup.
         </Serif>
       )}
-    </StepSummaryItem>
+    </StyledStepSummaryItem>
   )
 }
 
+const StyledStepSummaryItem = styled(StepSummaryItem)`
+  ${media.xs`
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+  `};
+`
```

Comment:
This is the right approach, but these components are used in different places and not always together, so these border changes are appearing in places they shouldn't. e.g. here's how the payment page looks with these changes:

![image](https://user-images.githubusercontent.com/1242537/55346346-8abc0300-54b2-11e9-8e82-a3162113d2cb.png)

We can move these `media.xs` styles up into the status page by styling the `ShippingSummaryItem` and `TransactionDetailsSummaryItem` in `Status/index.tsx`. 

e.g.

```ts
const StyledShippingSummaryItem = styled(ShippingSummaryItem)`
  ${media.xs`
    border-top-left-radius: 0;
    border-top-right-radius: 0;
  `};
`
```